Source/ACE.Server/WorldObjects/WorldObject_Magic.cs

```diff
@@ -955,7 +955,6 @@ protected bool SummonPortal(uint portalId, Position location, double portalLifet
             gateway.MinLevel = portal.MinLevel;
             gateway.MaxLevel = portal.MaxLevel;
             gateway.PortalRestrictions = portal.PortalRestrictions;
-            gateway.PortalRestrictions |= PortalBitmask.NoSummon;
```

Comment:
any reference for this?

Reply:
none, and in fact, I believe it to be an incorrect memory.. adjusted PR